bin/prettier.js

```diff
@@ -141,7 +141,9 @@ function format(input) {
     const pp = prettier.format(input, options);
     const pppp = prettier.format(pp, options);
     if (pp !== pppp) {
-      const diff = require('diff').createTwoFilesPatch('', '', pp, pppp, '', '', {context: 2});
+      const diff = require(
```

Comment:
I think it feels weird because you're used to seeing `require(lib)` as an atomic unit. It it were something like `myObject.method` I don't think it would stand out as much.

:man_shrugging: I'm fine accepting it :)